sdk/cognitivelanguage/ai-language-text/src/lro.ts

```diff
@@ -266,12 +266,31 @@ type Writable<T> = {
  */
 export function createUpdateAnalyzeState(docIds?: string[]) {
   return (state: AnalyzeBatchOperationState, lastResponse: LroResponse): void => {
-    const { createdOn, modifiedOn, id, displayName, expiresOn, tasks, lastUpdateDateTime } =
-      lastResponse.flatResponse as AnalyzeTextJobStatusResponse & { lastUpdateDateTime: string };
+    const {
+      createdOn,
+      modifiedOn,
+      id,
+      displayName,
+      expiresOn,
+      tasks,
+      lastUpdateDateTime,
+      status,
+      errors,
+    } = lastResponse.flatResponse as AnalyzeTextJobStatusResponse & { lastUpdateDateTime: string };
     const mutableState = state as Writable<AnalyzeBatchOperationState> & {
       docIds?: string[];
     };
     mutableState.createdOn = createdOn;
+    if (status.toLowerCase() === "partiallysucceeded") {
+      if (!errors) {
+        mutableState.status = "succeeded";
+      } else {
```

Comment:
@kristapratico how does the python client handle the case where the status is partiallysucceeded but there are no errors list?

Reply:
I've never seen the service return partiallySucceeded without errors -- is this something you're seeing?

Reply:
No but something we have to handle because the errors list could be undefined according to how it is typed. I just wanted to align the handling in both languages just in case.

Reply:
I see. I think I'd handle it similarly as the "no error targets provided" scenario, i.e. raise an exception.

Reply:
That makes sense, it is a choice between trusting the status or the errors list and I think trusting the status makes more sense. Hopefully that situation never happens but still it is good to be prepared. @minhanh-phan could you please error in this case too?

Reply:
That makes sense. I'll make an error with a message with "no errors returned"